.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn commitlint --edit $1
+npx commitlint --edit $1
```

Comment:
You're going to need to run the chmod cmd to update the hook if you change this

Reply:
Wdym? It still has the same execute permissions?